calcom/cal.com
# feat: Org owners should be able to see all bookings in bookings tab.

## What does this PR do?



- Fixes https://github.com/calcom/cal.com/issues/15302
- Fixes CAL-3852



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- Create an org with subteams and more people, book events in subteams as well as in private people calendars, you should see all of them in the bookings tab given that you are either an `ADMIN` or  `OWNER` role within the org.
